src/app/app.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [AppComponent],
+    }).compileComponents();
+  });
+
+  it('should create the app', () => {
+    const fixture = TestBed.createComponent(AppComponent);
```

Comment:
issue

          declarations: [AppComponent],

          Explantion: The 'AppComponent' should be declared in declarations instead of imports when configuring testing module in Angular.